settings.py

```diff
@@ -1,35 +1,29 @@
 import os
 from enum import unique, Enum
 
-ARN_ID = "arn:aws:lambda:us-west-1:<your lambda id>:function:update-slack-oncall"
 
 #TODO move these to a settings.yml so they can be re-used in the serverless yml
 # using https://serverless.com/framework/docs/providers/aws/guide/variables/#reference-variables-in-other-files
 
+STAGE = os.environ['stage']
 
 @unique
 class People(Enum):
     """
     <configure-me>
-    person1 = 'email1@email.com'
-    person2 = 'email2@email.com'
+    person1 = ('email1@email.com', phonenumber1)
```

Comment:
juuuuust make it a Person class then i hate value[0], value[1]

Reply:
FINE